R/parallel_new.R

```diff
@@ -0,0 +1,205 @@
+# This are new changes for parallel process via future _or_ mirai. These
+# functions should be able to replace what is contained in `parallel.R`
+
+# ------------------------------------------------------------------------------
+
+has_non_par_pkgs <- function(object, control, verbose = FALSE) {
```

Comment:
I don't think that we'll expose `verbose` to users but we might want it internally.